pkg/reconciler/sequence/sequence.go

```diff
@@ -385,7 +386,7 @@ func (r *Reconciler) reconcileEventPolicies(ctx context.Context, s *v1.Sequence,
 	for _, inputPolicy := range inputPolicies {
 		existingInputPolicy, exists := existingPolicyMap[inputPolicy.Name]
 		if exists {
-			if !equality.Semantic.DeepDerivative(inputPolicy, existingInputPolicy) {
+			if !reflect.DeepEqual(inputPolicy, existingInputPolicy) {
```

Comment:
Why was this changed from `equality.Semantic.DeepDerivative`? `DeepDerivative(inputPolicy, existingInputPolicy)` ignores empty fields in `inputPolicy`, which can be useful, as `existingInputPolicy` probably has some updated `.status` or annotations added by k8s (e.g. last-applied-configuration annotation).